### ISSUE TITLE
After entering a new record, the database is not saved

### ISSUE THREAD

**Issue by ole77** [keepassxreboot/keepassxc#1887]
After entering a new record, the database is not saved

## Expected Behavior
When you try to save a database after you type a new record, 
the database's database file is deleted.

## Current Behavior
If describing a bug, tell us what happens instead of the expected behavior
If suggesting a change/improvement, explain the difference from the current behavior

## Steps to Reproduce (for bugs)
1. Clone entry
2. Ctrl+S
3. Displays a message:
     "Writing the database failed.
     Cannot create /run/user/1000/doc/249ecea6/Olk.kdbx for output"

## Debug Info
KeePassXC - Version 2.3.1
Revision: 2fcaeea
Distribution: Flatpak

Libraries:
- Qt 5.10.1
- libgcrypt 1.7.3

Operating system: Linux 4.10.0-38-generic
CPU architecture: x86_64
Kernel: linux 4.10.0-38-generic

Enabled extensions:
- Auto-Type
- Browser Integration
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey

**Comment by droidmonkey** [comment]
Are you intentionally trying to save your database in the run folder? You can't do that

**Comment by ole77** [comment]
No, not a hint. If after entering a new entry, I press Ctrl + Q - the same message is output. The same file opened in KeepassXC under in Windows 10 is working fine

**Comment by droidmonkey** [comment]
What I'm saying is the error message indicates your database file is stored or trying to be stored in the /run folder.

**Comment by ole77** [comment]
The thing is that when I selected the .kdbx file in the password input window, the path \run\user\...

**Comment by zderi** [comment]
I have the same problem. When I open database from, for example, Documents folder my database is transferred to /run folder. When I check out Documents folder the database is no longer there and of course I can't save the database.
